scivision/catalog/data/datasources.json

```diff
@@ -1,46 +1,85 @@
 {
-  "data-000":{
-    "task":["object-detection", "segmentation"],
-    "domain":["optical-microscopy"],
-    "datasource":"https://github.com/stardist/stardist/releases/download/0.3.0/demo3D.zip",
-    "format":"tif",
-    "labels":"yes",
-    "institution":"epfl",
-    "tags":[
-      "help-needed", "3D", "cell", "cell-counting", "biology", "biomedical-science" 
-    ]
-  },
-  "data-001":{
-    "task":["segmentation", "thresholding", "object-detection"],
-    "domain":["multispectral", "hyperspectral"],
-    "datasource":"https://gitlab.au.dk/AUENG-Vision/OPPD/-/archive/master/OPPD-master.zip?path=DATA/images_plants/1COMF",
-    "format":"jpg",
-    "labels":"yes",
-    "institution":"Aarhus University",
-    "tags":[
-      "help-needed", "2D", "plant-phenotyping", "agriculture", "climate-change-and-agriculture"
-    ]
-  },
-  "data-002":{
-    "task":["classification"],
-    "domain":["oceanography"],
-    "datasource":"https://github.com/scotthosking/intake-plankton/blob/main/plktn.yaml",
-    "format":"intake",
-    "labels":"yes",
-    "institution":"CEFAS",
-    "tags":[
-      "help-needed", "2D" 
-    ]
-  },
-  "data-003":{
-    "task":["object-detection"],
-    "domain":["computer-vision"],
-    "datasource":"https://upload.wikimedia.org/wikipedia/commons/thumb/2/21/Cutest_Koala.jpg/262px-Cutest_Koala.jpg",
-    "format":"jpg",
-    "labels":"yes",
-    "institution":"Wikipedia by Erik Veland",
-    "tags":[
-      "demo"
-    ]
-  }
+   "catalog_type":"scivision datasource catalog",
+   "name":"default",
+   "entries":[
+      {
+         "name":"data-000",
```

Comment:
@ots22 how do you feel about this naming convention? Could we have the user submitting the entry give it a name? Perhaps a `dataset_id` or `model_id` field additionally that's an integer? Then maybe a primary key that's a combination of the two or something? Might make it easier to move from json to SQL at a later date if that's ever the plan

Reply:
Sorry by the naming convention I mean the name under entries e.g. "data-000"

Reply:
I was thinking that the name is the key, and we enforce uniqueness via an automated check. I don't think there's a need to stick to a particular naming convention (like 'data-NNN') and the names could be descriptive. We wouldn't then need an 'id' field or anything like that (although the dataframe gets a numerical index anyway).

Reply:
yeah, that's how I did on my other branch with the `add` functions